.github/PULL_REQUEST_TEMPLAE.md

```diff
@@ -0,0 +1,11 @@
+| Q             | A
+| ------------- | ---
+| Branch?       | "master" for new features / 1.0, 1.1, 1.2, 1.3, 1.4 for fixes
```

Comment:
i would want to avoid listing all branches here. we would need to maintain that constantly. rather say "the branch of the current release". (also, we don't maintain 1.0 anymore, bugfixes would go into 1.2 atm)